samples/place-autocomplete-basic-map/index.ts

```diff
@@ -12,50 +12,54 @@ const placeDetailsElement = document.querySelector(
   'gmp-place-details-compact',
 ) as any;
 const placeDetailsParent = placeDetailsElement.parentElement as HTMLElement;
-const mapDiv = document.getElementById('map-container') as HTMLElement;
-const center: google.maps.LatLngLiteral = { lat: 40.749933, lng: -73.98633 }; // New York City
-
+const gmpMapElement = document.querySelector('gmp-map') as any;
 
 async function initMap(): Promise<void> {
   // Asynchronously load required libraries from the Google Maps JS API.
   await google.maps.importLibrary('places');
-  const { AdvancedMarkerElement } = await google.maps.importLibrary('marker') as typeof google.maps.marker;
-  const { Map, InfoWindow } = await google.maps.importLibrary('maps') as any;
+  const {AdvancedMarkerElement} = (await google.maps.importLibrary(
+    'marker',
+  )) as google.maps.MarkerLibrary;
+  const {InfoWindow} = (await google.maps.importLibrary(
+    'maps',
+  )) as google.maps.MapsLibrary;
+
+  // Get the initial center directly from the gmp-map element's property.
+  const center = gmpMapElement.center;
 
   // Set the initial location bias for the autocomplete element.
   placeAutocompleteElement.locationBias = center;
 
   // Create the map object with specified options.
-  const map: google.maps.Map = new Map(mapDiv, {
-    zoom: 12,
-    center: center,
-    mapId: 'DEMO_MAP_ID',
+  const map = gmpMapElement.innerMap;
+  map.setOptions({
     clickableIcons: false,
     mapTypeControl: false,
     streetViewControl: false,
   });
 
   // Create an advanced marker to show the location of a selected place.
-  const advancedMarkerElement: google.maps.marker.AdvancedMarkerElement = new AdvancedMarkerElement({
-    map: map,
-  });
+  const advancedMarkerElement: google.maps.marker.AdvancedMarkerElement =
```

Comment:
one the `any`s are gone above, I think you can strip out all the `: type` type decls?